src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h

```diff
@@ -98,27 +98,19 @@ namespace Util
          */
         void publishAndClearLayers();
 
-        /**
-         * Clears the content (shape vector) in the layer message upon method call
-         */
-        void clearLayers();
-
         // Drawing methods
         /**
-         * Request a message to draw a ellipse shape. The origin is the center of the
-         * circle.
-         *
-         * @param layer: The layer name this shape is being drawn to
-         * @param cx: Ellipse's center X
-         * @param cy: Ellipse's center Y
-         * @param r1: Ellipse's horizontal radius
-         * @param r2: Ellipse's vertical radius
-         * @param draw_style: the drawing style of the shape
-         * @param draw_transform: the transformation of the shape
-         */
-        void drawEllipse(const std::string& layer, double cx, double cy, double r1,
-                         double r2, DrawStyle draw_style = DrawStyle(),
-                         DrawTransform draw_transform = DrawTransform());
+        * Request a message to draw a ellipse shape. The origin is the center of the
+        * circle.
+        *
+        * @param layer: The layer name this shape is being drawn to
+        * @param cx: Ellipse's center X
+        * @param cy: Ellipse's center Y
```

Comment:
units? (should be mm I think)

This goes for all the params for each shape.

Reply:
Will add missing units in latest commit